addons/mass_mailing/models/mailing_list.py

```diff
@@ -126,6 +126,33 @@ def copy(self, default=None):
     # ACTIONS
     # ------------------------------------------------------
 
+    def action_open_import(self):
+        """Open the mailing list contact import wizard."""
+        self.ensure_one()
+        action = self.env['ir.actions.actions']._for_xml_id('mass_mailing.mailing_contact_import_action')
+        action['context'] = {
```

Comment:
Below you seem to keep the current context but here it's overridden.
Not sure it's a big deal but would be nice to keep consistency ^^